dhis-2/dhis-support/dhis-support-hibernate/src/main/java/org/hisp/dhis/dbms/HibernateDbmsManager.java

```diff
@@ -323,7 +323,7 @@ public void emptyDatabase() {
     emptyTable("previouspasswords");
     emptyTable("usersetting");
     emptyTable("fileresource");
-    emptyTable("trackedentitycomment");
+    emptyTable("note");
```

Comment:
might make sense to move this table close to the related tables `eventnotes`, `enrollmentnotes` so they are grouped